backend/controllers/user.js

```diff
@@ -1,6 +1,19 @@
 const User = require("./../models/User");
 const bcrypt = require("bcrypt");
 
+const isEmailUnique = async (emailId) => {
```

Comment:
Do we really need this?
If we are not doing runtime checks we can just rely on the unique constraint enforced on the database.

Reply:
It might be difficult to say if the error was because of email or handle when creating a user right?